conf/messages

```diff
@@ -1,5 +1,5 @@
-error.postcode.invalid = Please enter a valid postcode
-error.postcode.blacklisted = This postcode is blocked and cannot be used
+error.postcode.invalid = You have entered an invalid postcode
+error.postcode.blacklisted = You can't use the postcode you've entered
```

Comment:
I changed the error messages according to this document: https://docs.google.com/spreadsheets/d/1gIEJJecD_EjBgDlISAcXbpNfQRCIP-SKM5qbdYo7_fQ/edit#gid=0, (subscription-address tab).